client/my-sites/importer/importing-pane.jsx

```diff
@@ -149,20 +149,20 @@ export const ImportingPane = React.createClass( {
 			);
 		}
 
-		return this.translate( 'Import complete!' );
+		return translate( 'Import complete!' );
 	},
 
 	getImportMessage( numResources ) {
 		if ( 0 === numResources ) {
-			return this.translate( 'Finishing up the import' );
+			return translate( 'Finishing up the import' );
 		}
 
-		return this.translate(
-			'Waiting on %(numResources)d resource to import',
-			'Waiting on %(numResources)d resources to import',
+		return translate(
+			'Waiting on %(numResources)s resource to import',
```

Comment:
In one of the comments on Medium importer feedback it was pointed out that this message is not user-friendly, and they've suggested to change it to `Importing X items` or similar. What do you think?